### PR TITLE
[DO-1729] Finished rule 10 IsMuchParTimeJob

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/pep_scoring/rules.py [hunk 1]
```diff
@@ -356,6 +356,55 @@ def calculate_weight(self) -> Tuple[Union[int, float], dict]:
         return RESULT_FALSE
 
 
+@register_rule
+class IsMuchParTimeJob(BaseScoringRule):
```
Comment (criticus) [inline business_register/pep_scoring/rules.py:360]: is it PartTime job? typo?

**Review by criticus — APPROVED** [review]
looks good to me

File: business_register/pep_scoring/rules.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/pep_scoring/rules.py:360]: Yeah, thank you! A bit tired))

Comment (IegorSoboliev) [reply business_register/pep_scoring/rules.py]: fixed

**Review by Andrii17s — APPROVED** [review]

**Pull request merged**